daemons/build_deploy_config.sh

```diff
@@ -13,6 +13,7 @@ if [[ -z $DSS_DEPLOYMENT_STAGE ]]; then
 fi
 
 export daemon_name=$1
+export daemon_tag=$(echo $daemon_name | cut -d '-' -f2-)
```

Comment:
Why do you edit the daemon name instead of just using `daemon_name`?

Reply:
its to strip out dss from the name to avoid `dss-dss-operations-dev` as the Name.